dev-lang/python/patches/python3.10-3.10.13.patchset

```diff
@@ -1077,3 +1077,117 @@ index 57c84e5..6d4f5fc 100644
 -- 
 2.42.1
 
+
+From 5fa26ebf8065b6d0912f3c039985df339856c9cc Mon Sep 17 00:00:00 2001
+From: Oscar Lesta <oscar.lesta@gmail.com>
+Date: Sat, 10 Feb 2024 06:01:25 -0300
+Subject: Fix build on nightlies, following the addition of kqueue.
+
+Together with the patch for gh-109191, fixes HaikuPorts issue #10001.
+
+The fix for test_kqueue is just a quick hack, but good enough for now.
+
+diff --git a/Lib/test/test_kqueue.py b/Lib/test/test_kqueue.py
+index 998fd9d..8aaea06 100644
+--- a/Lib/test/test_kqueue.py
++++ b/Lib/test/test_kqueue.py
+@@ -127,19 +127,19 @@ class TestKQueue(unittest.TestCase):
+ 
+         ev = select.kevent(server.fileno(),
+                            select.KQ_FILTER_WRITE,
+-                           select.KQ_EV_ADD | select.KQ_EV_ENABLE)
++                           select.KQ_EV_ADD)
```

Comment:
the correct way would be to defined select.KQ_EV_ENABLE to zero, don't you think?